build.sh

```diff
@@ -164,18 +195,24 @@ if [ -f "${OUTISO}" ]; then
 fi
 
 # create iso
-make iso BASE="${MYBASE}" RELEASE="${MYRELEASE}" ARCH="${MYARCH}" ROOTPW_HASH="*"
+if [ -n "${UPLOAD}" ]; then
+    make iso BASE="${MYBASE}" RELEASE="${MYRELEASE}" ARCH="${MYARCH}" ROOTPW_HASH="*"
+else
+    exit_error "UPLOAD is unset"
+fi
```

Comment:
```suggestion
make iso BASE="${MYBASE}" RELEASE="${MYRELEASE}" ARCH="${MYARCH}" ROOTPW_HASH="*"
```

(assuming UPLOAD is NO by default as suggested above)